node/network/protocol/src/lib.rs

```diff
@@ -17,7 +17,7 @@
 //! Network protocol types for parachains.
 
 #![deny(unused_crate_dependencies)]
-#![warn(missing_docs)]
+#![warn(missing_docs, unused_imports)]
```

Comment:
aren't they on by default?

Reply:
Yeah they are, just had problems with not seeing them. Forgot to remove that again :-)